app/forms/vehicle_details_form.rb

```diff
@@ -26,4 +27,6 @@ class VehicleDetailsForm
 
   attribute :vehicle_in_regular_use, :boolean
   validates :vehicle_in_regular_use, inclusion: { in: [true, false], allow_nil: false }
+
+  attribute :vehicle_in_dispute, :boolean
```

Comment:
It would nice to have a boolean validation to match the others. Got bitten by this in a previous project when the data started coming from another source...